assets/terraform/gce/os.tf

```diff
@@ -10,17 +10,20 @@ variable "oss" {
 
   default = {
     # os -> {project}/{image}
-    "ubuntu:16"     = "ubuntu-os-cloud/ubuntu-1604-xenial-v20190816"
-    "ubuntu:18"     = "ubuntu-os-cloud/ubuntu-1804-bionic-v20190813a"
-    "ubuntu:latest" = "ubuntu-os-cloud/ubuntu-1804-bionic-v20190813a"
-    "redhat:7"      = "rhel-cloud/rhel-7-v20190813"
-    "centos:7"      = "centos-cloud/centos-7-v20190813"
-    "debian:8"      = "debian-cloud/debian-8-jessie-v20180401"
-    "debian:9"      = "debian-cloud/debian-9-stretch-v20190813"
-    "debian:latest" = "debian-cloud/debian-9-stretch-v20190813"
-    "suse:12"       = "suse-cloud/sles-12-sp4-v20190617"
-    "suse:15"       = "suse-cloud/sles-15-sp1-v20190828"
-    "suse:latest"   = "suse-cloud/sles-15-sp1-v20190828"
+    "ubuntu:16"     = "ubuntu-os-cloud/ubuntu-1604-xenial-v20200407"
+    "ubuntu:18"     = "ubuntu-os-cloud/ubuntu-1804-bionic-v20200414"
+    "ubuntu:19"     = "ubuntu-os-cloud/ubuntu-1904-eoan-v20200413a"
+    "ubuntu:latest" = "ubuntu-os-cloud/ubuntu-1904-eoan-v20200413a"
+    "redhat:7"      = "rhel-cloud/rhel-7-v20200420"
+    "redhat:8"      = "rhel-cloud/rhel-8-v20200413"
+    "centos:7"      = "centos-cloud/centos-7-v20200420"
+    "centos:8"      = "centos-cloud/centos-8-v20200413"
+    "debian:9"      = "debian-cloud/debian-9-stretch-v20200420"
```

Comment:
We say Gravity supports Debian 8 here:  https://gravitational.com/gravity/docs/requirements/  What is your thinking behind removing it?  I recognize we don't run those tests presently, and I'd like to figure out if we should either:

1) plan to test it (https://github.com/gravitational/robotest/issues/182)
2) remove it from the list of distros that Gravity supports (would that be an 8.0 thing given semver?)

Reply:
Yes, I fell into a trap of thinking google is no longer supporting the `jessie`, but it does show up under deprecated. I'll put it back in.